lib/rubocop/cop/g2/ban_sidekiq_worker.rb

```diff
@@ -0,0 +1,18 @@
+module RuboCop
+  module Cop
+    module G2
+      class BanSidekiqWorker < RuboCop::Cop::Cop
+        MSG = 'Please use Sidekiq::Job instead of Sidekiq::Worker'.freeze
+
+        def_node_search :sidekiq_worker?, <<~PATTERN
+          (send nil? :include (const (const nil? :Sidekiq) :Worker))
+        PATTERN
+
+        def on_class(node)
```

Comment:
This is a minor nitpick but do you mind changing this to `on_send` instead?
```suggestion
        def on_send(node)
```
I think it's a bit nicer for error highlighting because it doesn't highlight the class keyword the the class name, just the `include` statement.

Reply:
Nice, I'll do change it.